src/array_api_stubs/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import _2021_12, _2022_12, _draft
```

Comment:
If the idea is that these are public submodules, I don't think we should start them with underscores.

Reply:
Ah so I've left stubs packaging to a follow-up PR (see collapsed Follow-up PRs section in the submission post). I wouldn't want these modules public anywho, instead opting for an API more aligned with #472.

Reply:
Why don't we want the stubs modules public?